athena-vertica/athena-vertica.yaml

```diff
@@ -2,15 +2,15 @@ Transform: 'AWS::Serverless-2016-10-31'
 
 Metadata:
   AWS::ServerlessRepo::Application:
-    Name: VerticaAthenaConnector
-    Description: verticaAthenaConnector Description
+    Name: AthenaVerticaConnector
+    Description: 'This connector enables Amazon Athena to communicate with Vertica'
     Author: 'default author'
     SpdxLicenseId: Apache-2.0
     LicenseUrl: LICENSE.txt
     ReadmeUrl: README.md
     Labels: ['athena-federation']
     HomePageUrl: https://github.com/awslabs/aws-athena-query-federation
-    SemanticVersion: 2021.35.10
+    SemanticVersion: 2021.18.1
```

Comment:
Looks like the version was added incorrectly a few releases back, so it wasn't updated when doing a find and replace.